tests/wpt/metadata-css/css21_dev/html4/block-in-inline-remove-000.htm.ini

```diff
@@ -1,3 +1,3 @@
 [block-in-inline-remove-000.htm]
   type: reftest
-  expected: FAIL
+  expected: PASS
```

Comment:
Isn't the norm to just delete the file now, and the testdriver will assume a pass is expected?